assets/components/whySupport/whySupport.scss

```diff
@@ -0,0 +1,350 @@
+      margin-left: -50px;
+    }
+
+    @include mq($from: mobileLandscape) {
+      width: gu-breakpoint(mobileLandscape);
+      margin-left: 0;
+    }
+
+    @include mq($from: phablet) {
+      width: gu-breakpoint(phablet);
+    }
+
+    @include mq($from: tablet) {
+      width: gu-breakpoint(tablet) + 120px;
+      margin-left: - ((gu-breakpoint(tablet) + 120px) - gu-breakpoint(tablet)) / 2;
+    }
+
+    @include mq($from: desktop) {
+      display: none;
+    }
+  }
+
+  .svg-wall-desktop {
+    display: none;
+
+    @include mq($from: desktop) {
+      display: block;
+      width: gu-breakpoint(wide);
+      margin-left: - (gu-breakpoint(wide) - gu-breakpoint(desktop)) / 2;
+    }
+
+    @include mq($from: leftCol) {
+      margin-left: - (gu-breakpoint(wide) - gu-breakpoint(leftCol)) / 2;
+    }
+
+    @include mq($from: wide) {
+      margin-left: 0;
+    }
+  }
+}
+
+.component-why-support__floating-circle {
+  border-radius: 600px;
+  background-color: gu-colour(multimedia-main-2);
+  width: 50px;
+  height: 50px;
+
+  @include mq($from: tablet) {
+    width: 60px;
+    height: 60px;
+  }
+}
+
+.component-why-support__heading {
+  font-family: $gu-egyptian-web;
+  font-weight: bold;
+  font-size: 30px;
+  line-height: 34px;
+  color: gu-colour(neutral-1);
+
+  @include mq($from: mobileMedium) {
+    font-size: 36px;
+    line-height: 36px;
+  }
+
+  @include mq($from: tablet) {
+    font-size: 60px;
+    line-height: 60px;
+  }
+}
+
+.component-why-support__subheading {
+  display: block;
+}
+
+.component-why-support__heading--edits {
+  position: relative;
+  margin-top: $gu-v-spacing * 3;
+  height: 180px;
+
+  @include mq($from: mobileLandscape) {
+    height: 200px;
+  }
+
+  @include mq($from: phablet) {
+    height: 300px;
+  }
+
+  @include mq($from: phablet) {
+    padding-left: 180px;
+  }
+
+  @include mq($from: tablet) {
+    padding-left: 140px;
+  }
+
+  &::after {
+    @extend .component-why-support__floating-circle;
+    content: " ";
+    position: absolute;
+    left: 230px;
+    bottom: $gu-v-spacing;
+
+    @include mq($from: phablet) {
+      left: 170px;
+      bottom: $gu-v-spacing * 3;
+    }
+  }
+}
+
+.component-why-support__subheading--edits-indented {
+  margin-left: 50px;
+
+  @include mq($from: tablet) {
+    margin-left: 95px;
+  }
+}
+
+.component-why-support__heading-text-wrapper {
+  position: absolute;
+  top: 26px;
+  left: 48px;
+
+  @include mq($from: mobileMedium) {
+    top: 32px;
+    left: 60px;
+  }
+
+  @include mq($from: mobileLandscape) {
+    top: 50px;
+    left: 100px;
+  }
+
+  @include mq($from: phablet) {
+    top: 80px;
+    left: unset;
+  }
+}
+
+.component-why-support__heading--advertising {
+  padding: $gu-v-spacing 0 150px;
+  margin-top: $gu-v-spacing * 4;
+  position: relative;
+
+  @include mq($from: phablet) {
+    padding: $gu-v-spacing 0 152px 80px;
+  }
+
+  @include mq($from: tablet) {
+    padding-bottom: 132px;
+  }
+
+  &::after {
```

Comment:
Is it possible to avoid nesting?

Reply:
Flattening the `after`? That would look like this:

```scss
.component-why-support__heading--advertising::after {
  @extend .component-why-support__floating-circle;
  content: " ";
  position: absolute;
  bottom: $gu-v-spacing;
  left: 250px;

  @include mq($from: phablet) {
    right: 0;
    left: unset;
  }
}
```

However, I'm not sure I see the benefit of that, as both versions of writing this out have the same specificity. In terms of readability I prefer to use the `&::after` version because the pseudo-element is tied to the class regardless, and it means there are fewer places to look for rules relating to that class.